baseplate/experiments/providers/legacy.py

```diff
@@ -0,0 +1,233 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import hashlib
+
+from .base import ExperimentInterface
+from ..._compat import long, iteritems
+
+
+logger = logging.getLogger(__name__)
+
+
+class LegacyExperiment(ExperimentInterface):
```

Comment:
can we call this something more descriptive of when it'd be preferred over an alternative (which we don't really have yet right?) maybe R2Experiment? 

(bringing this up since i got similar feedback recently: "legacy" got rightfully called out as vague and not informative when doing docs for the memcache serialization formats.)